class-lab/exercise3/car.js

```diff
@@ -0,0 +1,37 @@
+class Car {
+  constructor(color, speed = 0) {
+    this.color = color;
+    this.speed = speed;
+  }
+
+  drive(speed) {
+    this.speed = speed;
+  }
+
+  isFasterThan(other) {
+    return this.speed > other.speed;
+  }
+}
+
+class Race {
+  cars = [];
+
+  constructor() {
+  }
+
+  addCars(c1, c2, c3) {
+    this.cars[0] = c1;
+    this.cars[1] = c2;
+    this.cars[2] = c3;
+  }
```

Comment:
If you already have learned about the spread operator, you can define a function that has an unlimited number of arguments, like this:
```js
addCars(...cars) {
  this.cars = [...cars]
}
```

Reply:
changes, Thanks!